pages/common/siege.md

```diff
@@ -0,0 +1,28 @@
+# siege
+
+> HTTP loadtesting and benchmarking tool.
+> More Information: <https://www.joedog.org/siege-manual/>.
+
+- Test a url with default settings:
+
+`siege https://www.google.com`
+
+- Test a list of urls:
+
+`siege --file /path/to/url-list.txt`
```

Comment:
Perhaps this would be more consistent with other existing pages?
```suggestion
`siege --file {{path/to/url-list.txt}}`
```

Reply:
Yup, updated in latest commit